Backoffice/LeftPanel/Strategies/AdministrationPanelStrategy.php

```diff
@@ -79,4 +80,24 @@ public function getRole()
     {
         return $this->role;
     }
+
+    /**
+     * @return string
+     */
+    protected function getTemplate()
+    {
+        if ($this->template) {
```

Comment:
I will do an issue for this ticket, I think there need to be a more global reflexion about that.

But I agree with you on this point